### ISSUE TITLE
t.w.http.Request docstrings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2752]
|<img alt="rikyu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| rikyu reported|
|-|-|
|Trac ID|trac#2752|
|Type|enhancement|
|Created|2007-07-26 17:23:22Z|

Added a couple of missing docstrings for http.Request

Attachments:

* [http.py.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/fa9/fa92c3321aa5b75262b545f02d3fc30a058171b3/de671f4e1a1c7ad95bd2e9d52024877656afaa1b.diff) (2182 bytes) - added by rikyu on 2007-07-26 17:24:05Z - twisted.web.http docstring diff

<details><summary>Searchable metadata</summary>

```
trac-id__2752 2752
type__enhancement enhancement
reporter__rikyu rikyu
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1185470602000000 1185470602000000
changetime__1185479890000000 1185479890000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve set owner to @therve|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set status to `closed`|
|-|-|

(In [20836]) Add some docstrings to methods of twisted.web.http.Request, with a couple of
cleanups.

Author: rikyu
Reviewer: therve
Fixes [#2752](https://github.com/twisted/twisted/issues/2752)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
